_benchmark/workloads/indices.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: indices
+parent: Workload reference
+nav_order: 65
+---
+
+The `indices` element contains a list of all indices used in the workload. 
+
+## Example
+
+```json
+"indices": [
+    {
+      "name": "geonames",
+      "body": "geonames-index.json",
+    }
+]
+```
+
+## Configuration options
+
+Use the following options with `indices`:
+
+Parameter | Required | Type | Description
+:--- | :--- | :--- | :---
+| `name` | Yes | String | The name of the index template. |
+| `body` | No | String | The file name corresponding to the index definition that is used in the body of the create index API. |
```

Comment:
Should the name of the API be capitalized?

Reply:
Done.